### ISSUE TITLE
Can't find zip file

### ISSUE THREAD

**Issue by gyrocn** [PaulLereverend/NextcloudExtract#39]
I try extra a zip file in extra store，then alter me this."can't find zip file".
I try a other a file, in normal store, it's works.
but i wan't extra zip file in extra store.
Hope it can get fixed soon! :)
I use "External storage support"  apps on nextcloud 15.0.11.

**Comment by gyrocn** [comment]
![image](https://user-images.githubusercontent.com/18325639/65485776-54959400-ded5-11e9-99c3-d2d9dd1be120.png)

**Comment by PaulLereverend** [comment]
What type of external storage do you use (Samba, local, ftp...)  ? 

This app only supports local.

**Comment by ebuhl** [comment]
Hi, I have the same problem with local "Group Folders".
In own folder, is of course the group folder not visible, so we get the error.
Maybe it is not supported too...? :-(

**Comment by PaulLereverend** [comment]
Hi, 
Shared folders (to groups and users) are supported since the latest release. But unfortunately the app "Group folders" is not supported yet. 
While I work on this, may I suggest you extract the archive on your own folders and then copy the files in the group folder. 
I will keep you updated.

**Comment by Maxobus** [comment]
> What type of external storage do you use (Samba, local, ftp...) ?
> This app only supports local.

I have Local storage connected but have the same issue - "Can't find zip file".
When trying to unzip in root - it works.

Nexcloud 17.0.3 Stable.

Any ideas?

**Comment by PaulLereverend** [comment]
@Maxobus what is the path to your zip file ? (from the / directory) 

Is there any particular permissions on your folders that could cause this issue ?

**Comment by Maxobus** [comment]
In Linux it is: /NAS/FAMILY/PHOTOS
In Nextcloud: /NAS/PHOTOS

**Comment by PaulLereverend** [comment]
Is it working if you mount the storage on "/" in nextcloud ?

**Comment by Maxobus** [comment]
Yes, if zip file is placed in Nextcloud's default "/" (root) - everything is working.

**Comment by PaulLereverend** [comment]
Actually i meant mounting the external storage in nextcloud "/"  and not in "/NAS/"

**Comment by Maxobus** [comment]
I got some results.
If I mount external storage to root "/" or first level, fo example "working" it's working.
If I create a folder in root and then mount external storage to that folder - it doesn't work.

Here is a screenshot from External Storages plugin:
![Screen Shot 2020-02-03 at 15 53 19](https://user-images.githubusercontent.com/60475350/73658664-67974c80-469d-11ea-8b18-a9a58f67516d.png)

I have a zip file in "DOWNLOADS". Then I mounted it with two different paths.
As a result, first one is working, second - not.

![Screen Shot 2020-02-03 at 16 01 00](https://user-images.githubusercontent.com/60475350/73659191-64509080-469e-11ea-99c5-bac9f2e5bb12.png)

**Comment by PaulLereverend** [comment]
It looks like a regression. I'm going to look into it
